pkg/util/util.go

```diff
@@ -1061,6 +1062,11 @@ func ValidateURL(sourceURL string) error {
 // ValidateDockerfile validates the string passed through has a FROM on it's first non-whitespace/commented line
 // This function could be expanded to be a more viable linter
 func ValidateDockerfile(contents []byte) error {
+	s := log.Spinner("Validating Dockerfile")
+	if len(contents) == 0 {
+		s.End(false)
+		return errors.New("dockerfile URL provided in the Devfile does not point to an empty file")
```

Comment:
Error returned says "does not point to an empty file" but I think you meant "does point to an empty file" if the length is 0